### PR TITLE
Create Issue Template

### PR DESCRIPTION
**Pull request by seanpdoyle**
To increase the quality of both issue creation and issue triage,
introduce a [GitHub Issue Template][docs].

Once merged, issues that don't provide at least some of the context
requested by the template will likely be closed immediately until
additional information is provided.

[docs]: https://help.github.com/articles/creating-an-issue-template-for-your-repository/

### CODE REVIEW

**Comment by seanpdoyle** [comment]
cc: @thoughtbot/copy-editing

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**What operating system does the project run on?**
```
Comment (mike-burns) [inline .github/ISSUE_TEMPLATE.md]: "and version"?

Perhaps: "Which operating system and version is the project developed on?" is actually what we need?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**What is the [`ember-cli-rails`][gem] version (from `Gemfile`)?**
+
+**What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
+
+**What are the contents of `config/initializers/ember.rb`?**
+
+**Which are the contents of the Rails' view that renders the Ember application?**
+
+**How have are the EmberCLI-related routes defined?**
```
Comment (mike-burns) [inline .github/ISSUE_TEMPLATE.md]: Typo: "have are" should be "are".

**Comment by seanpdoyle** [comment]
@mike-burns I know you're not much of a JS'er, but does this look good to you?

**Comment by mike-burns** [comment]
LGTM!

**Pull request merged**